src/pepr/operator/controllers/keycloak/authservice/authorization-policy.ts

```diff
@@ -0,0 +1,127 @@
+import { K8s, Log } from "pepr";
+import { Action, AuthorizationPolicy, RequestAuthentication, UDSPackage } from "../../../crd";
+import { getOwnerRef } from "../../utils";
+import { Action as AuthServiceAction, AuthServiceEvent } from "./types";
+
+function authserviceAuthorizationPolicy(
+  labelSelector: { [key: string]: string },
+  name: string,
+  namespace: string,
+): AuthorizationPolicy {
+  return {
+    kind: "AuthorizationPolicy",
+    metadata: {
+      name: `${name}-authservice`,
+      namespace,
+    },
+    spec: {
+      action: Action.Custom,
+      provider: {
+        name: "authservice",
+      },
+      rules: [
+        {
+          when: [
+            {
+              key: "request.headers[authorization]",
+              notValues: ["*"],
```

Comment:
This policy allows bypassing Authservice (but not Istio's JWT verification) by including an `Authorization:` header in the incoming request.

If we have a real use case for this and we're confident in the Istio JWT verification (it needs to be improved), we _could_ keep this. Otherwise remove it.

Some possible reasons to keep this code are to allow:
* A mission app frontend may need to send a request to it's backend. Because the backend is authservice protected, this allows the frontend to send the user's JWT to the backend and pass Istio JWT valdiation
    *  I think this specific use case really makes a lot of sense./
    * Unless we expect people to remove authservice protection from mission app backend Pods?
* If any apps use something like the `grant_type=client_credentials` OAuth flow for (non-browser) API access to Authservice-protected mission apps.

Reply:
The https://github.com/defenseunicorns/uds-core/pull/201#discussion_r1643450893 fix should address this